### ISSUE TITLE
VM shuts down when easy-ocr is run on it

### ISSUE THREAD

**Issue by rishavmitra** [JaidedAI/EasyOCR#1203]
Hello,
We are trying to create a website that converts Images in a PDF to text. While running it locally this works works fine and after a minute we get the results.

But when I hosted my application on a VM in Azure, the VM shuts down whenever the `readtext()` function is called for reading large text filled images. There are no exceptions raised and no warnings.
### Here is a screenshot:
![image](https://github.com/JaidedAI/EasyOCR/assets/72710483/39d5d5c6-81f8-4e9d-824e-424b880ca8bd)

### This is the Easy-ocr function:
![image](https://github.com/JaidedAI/EasyOCR/assets/72710483/61c0d6a8-b7c5-4756-9d24-8cff455ca7e6)

Any help is highly appreciated.

**Comment by vishnupriyavr** [comment]
I am facing the same issue but on docker, any solution for this?

**Comment by rishavmitra** [comment]
> I am facing the same issue but on docker, any solution for this?

Yeah, the solution was upgrading your RAM, Seems like Easy OCR requires a minimum of 8GB ram to run. So bumping up the RAM on your VM might solve this issue.

**Comment by vishnupriyavr** [comment]
> > I am facing the same issue but on docker, any solution for this?
> 
> Yeah, the solution was upgrading your RAM, Seems like Easy OCR requires a minimum of 8GB ram to run. So bumping up the RAM on your VM might solve this issue.

Yeah you are right, figured it out a while ago.

**Comment by Siniara** [comment]
I'm having a similar issue. Running on an Azure GPU worker - 8 cores, 56 GB RAM, and 16GB vRAM. It just shuts down. :(

**Comment by daniellovera** [comment]
> I'm having a similar issue. Running on an Azure GPU worker - 8 cores, 56 GB RAM, and 16GB vRAM. It just shuts down. :( 

During training or inference?
Are you getting CUDA OOM errors?
Running the detection or recognition model? Or both?
